### ISSUE TITLE
Admin lite: Add Program disaggregations

### ISSUE THREAD

**Issue by jennyhavah** [mercycorps/TolaActivity#1720]
**Note:** This ticket only covers front end admin UI work in Admin Lite. It depends on #1208 back end work enabling program-specific disaggregations.

**UI detail:** Title Case Is Rampant In Admin Lite. I Created A Separate Ticket For This -- #1722.

# Issue

I have a place in Admin Lite to manage my program level disaggregations.

## Acceptance criteria

- [ ] 1. In Admin Lite, Programs section, each program has a _Program Disaggregations_ tab in between _Profile_ and _Settings_.

     ![image](https://user-images.githubusercontent.com/33670923/59952833-742ea380-9432-11e9-97ff-1438661ac5a4.png)

UI-wise, it works exactly like Country disaggregations. Including images below, mainly for acceptance testing purposes.

- [ ] 1. The empty state looks like this, swapping the word "program" for "country" in the title and "add a thing" button. <P>

     ![image](https://user-images.githubusercontent.com/33670923/59952929-dc7d8500-9432-11e9-9fd6-5aad36cbbc45.png)

- [ ] 2. After you click "Add program disaggregation"

     ![image](https://user-images.githubusercontent.com/33670923/59952918-cec7ff80-9432-11e9-94df-30949b36526e.png)

- [ ] 3. After you click "Save changes" -- this is really confusing:

     ![image](https://user-images.githubusercontent.com/33670923/59953630-de951300-9435-11e9-901e-c7f97ebfe5b9.png)

     I got disoriented and didn't know what to do next. What I would have preferred is if the form had stayed open, with the first Label field visible, like so:

     ![image](https://user-images.githubusercontent.com/33670923/59953665-03898600-9436-11e9-9dd2-b2944c4f3c09.png)

- [ ] 4. After typing my first label and clicking "Add another option"

     ![image](https://user-images.githubusercontent.com/33670923/59953719-35025180-9436-11e9-8314-011105536b8c.png)

- [ ] 5. I can remove labels _as long as they are not in use by any indicators._

## Housekeeping (where applicable)
- [ ] Unit test/s written 
- [ ] i18n strings included
- [ ] Is accessible to people using screen readers

**Comment by jennyhavah** [comment]
No longer needed -- going to explore a concept of archiving program-specific disaggs instead.
